src/ui/sass/modules/_FilterOptions.scss

```diff
@@ -170,7 +170,8 @@
       left: -20px;
       width: 3px;
       height: 9px;
-      transform: rotate(45deg) translateY(-50%);
+      transform: rotate(45deg) translateY(-50%) #{"/*rtl:rotate(45deg) translateX(-100%)*/"};
+      /*!rtl:ignore*/
```

Comment:
is this ignoring the border-right?

Reply:
yup

Reply:
For my own edification, why do we need to ignore the border for RTL?

Reply:
we want the right border to show instead of the left one so it's not inverted when we rotate/translate the symbol

Reply:
do we need to make it border-left for rtl then?

Reply:
ohh I see ignoring is different than removing, I thought this would just remove the line for rtl